apps/condo/domains/user/utils/testSchema/index.js

```diff
@@ -69,6 +72,28 @@ async function createTestUser (client, extraAttrs = {}, { raw = false } = {}) {
     return [result, attrs]
 }
 
+async function createTestUserExternalIdentity (client, extraAttrs = {}, { raw = false } = {}) {
+    if (!client) throw new Error('no client')
+    const sender = { dv: 1, fingerprint: 'test-' + faker.random.alphaNumeric(8) }
+    const identityId = faker.random.alphaNumeric(8)
+    const identityType = SBER_ID_IDP_TYPE
+    const meta = {
+        dv: 1, city: faker.address.city(), county: faker.address.county(),
+    }
+
+    const attrs = {
+        dv: 1,
+        sender,
+        identityId,
+        identityType,
+        meta,
+        ...extraAttrs,
+    }
+    const result = await UserExternalIdentity.create(client, attrs, { raw })
+    if (raw) return result
```

Comment:
You need to use throwIfError

Reply:
Since I use %EntityName%.create standard test method - this logic with 'throwIfError' already wrapped inside 'create' method